simpa_tests/manual_tests/generate_overview.py

```diff
@@ -0,0 +1,321 @@
+        if save_path == None:
+            self.save_path = os.path.join(self.current_dir, "figures/")
+        else:
+            self.save_path = save_path
+        if not os.path.isdir(self.save_path):
+            print(f"Created {self.save_path} directory")
+            os.mkdir(self.save_path)
+        self.ref_path = os.path.join(self.current_dir, "figures_ref/")
+        self.md_name = 'manual_tests_overview'
+        self.mdFile = MdUtils(file_name=self.md_name, title='<u>Overview of Manual Test Results</u>')
+        self.set_style()
+
+        # TODO fix issue in PointSourceReconstruction.py file (make it consistent with the other manual tests)
+        self.scripts_to_neglect = ["PointSourceReconstruction.py"]
+        # For testing this file one can ignore the time consuming scripts
+        """self.scripts_to_neglect += ["DelayAndSumReconstruction.py", 
+                                   "SignedDelayMultiplyAndSumReconstruction.py", "DelayMultiplyAndSumReconstruction.py",
+                                   "TimeReversalReconstruction.py", "MinimalKWaveTest.py", "ReproduceDISMeasurements.py",
+                                   "QPAIReconstruction.py", "TestLinearUnmixingVisual.py", "SegmentationLoader.py", "SimulationWithMSOTInvision.py",
+                                   "VisualiseDevices.py", "AbsorptionAndScatteringWithinHomogenousMedium.py", "ComputeDiffuseReflectance.py", 
+                                   "CompareMCXResultsWithDiffusionTheory.py", "AbsorptionAndScatteringWithInifinitesimalSlabExperiment.py"]"""
+        
+    def download_reference_images(self):
+        """
+        removes the current reference figures directory and downloads the latest references from nextcloud
+        """
+        ref_imgs_path = os.path.join(self.current_dir,"reference_figures")
+        if os.path.exists(ref_imgs_path):
+            # Remove the directory
+            shutil.rmtree(ref_imgs_path)
+            print(f'Directory {ref_imgs_path} removed successfully.')
+        # nextcloud url with the reference images
+        self.nextcloud_url = "https://hub.dkfz.de/s/Xb96SFXbmiE5Fk8" # shared "reference_figures" folder on nextcloud
+        # Specify the local directory to save the files
+        zip_filepath = os.path.join(self.current_dir, "downloaded.zip")#'downloaded_folder.zip')       
+        # Construct the download URL based on the public share link
+        download_url = self.nextcloud_url.replace('/s/', '/index.php/s/') + '/download'
+        # Send a GET request to download the file
+        response = requests.get(download_url)
+
+        if response.status_code == 200:
+            # Save the file
+            with open(zip_filepath, 'wb') as f:
+                f.write(response.content)
+            print(f'File downloaded successfully and stored at {zip_filepath}.')
+        else:
+            print(f'Failed to download file. Status code: {response.status_code}')
```

Comment:
Is this an error? Please use logging and raise an error

Reply:
I replaced all prints with logs using the simpa logger and added a raise error for this case